pkg/utils/helpers.go

```diff
@@ -196,3 +198,82 @@ func GetNounForm(noun string, length int) string {
 
 	return noun
 }
+
+func NewConfigHandler(filepath string) (ConfigHandler, error) {
+	fileExtension := fp.Ext(filepath)[1:]
+
+	switch fileExtension {
+	case "json":
+		return &JsonConfigHandler{Config{Filename: filepath}}, nil
+	case "yaml", "yml":
+		return &YamlConfigHandler{Config{Filename: filepath}}, nil
+	default:
+		return nil, errors.New("wrong file extension: JSON and YAML formats are accepted only")
```

Comment:
I think the error should be "wrong file extension: Only Json and Yaml formats are supported"